.gitignore

```diff
@@ -1,3 +1,8 @@
+### Shuffleboard ###
+
+# Ignore the test plugins jar, force it to be rebuilt locally
+app/src/test/resources/test_plugins.jar
```

Comment:
I'd put resources that are generated into the `/build` directory instead of doing this. Then gradle clean will clean these up.

Reply:
Do something like `/build/generated-resources`

Reply:
I just added `test_plugins/build/libs` as a test resource in the sourcesets. That seemed cleaner

Reply:
You should be able to remove this from the `.gitignore` now then right??

Reply:
Good catch!